optionals/compat_rhs_usf3/CfgWeapons.hpp

```diff
@@ -70,18 +70,19 @@ class CfgWeapons {
         ACE_barrelTwist = 177.8;
         ACE_barrelLength = 508.0;
     };
+    class rhs_weap_saw_base: Rifle_Base_F {
+        ACE_Overheating_allowSwapBarrel = 1;
+    };
     class rhs_weap_lmg_minimi_railed; // Rifle_Base_F - scope = private;
     class rhs_weap_m249_pip_S: rhs_weap_lmg_minimi_railed {
         ACE_RailHeightAboveBore = 4.11044;
         ACE_barrelLength = 348;
         ACE_barrelTwist = 177.8;
-        ACE_Overheating_allowSwapBarrel = 1;
```

Comment:
Why remove? do they inherit from saw_base? in that case the comment after `rhs_weap_lmg_minimi_railed` should be updated.

Reply:
inheritance is 
```
class rhs_weap_saw_base: Rifle_Base_F
class rhs_weap_lmg_minimipara: rhs_weap_saw_base
class rhs_weap_lmg_minimi_railed: rhs_weap_lmg_minimipara

//4 guns:
class rhs_weap_m249: rhs_weap_lmg_minimi_railed
class rhs_weap_m249_pip_S: rhs_weap_lmg_minimi_railed
class rhs_weap_m249_pip: rhs_weap_lmg_minimi_railed
class rhs_weap_m249_pip_L: rhs_weap_lmg_minimi_railed
```
I'll update comments